app/server/api-generator/src/main/java/io/syndesis/server/api/generator/swagger/util/JsonSchemaHelper.java

```diff
@@ -127,6 +132,34 @@ public static String serializeJson(final JsonNode schemaNode) {
         }
     }
 
+    public static JsonNode sanitize(final JsonNode node) {
+        if (node == null) {
+            return node;
+        }
+
+        if (node.isArray()) {
+            ArrayNode arrayNode = (ArrayNode) node;
+            Iterator<JsonNode> elements = arrayNode.elements();
+            elements.forEachRemaining(JsonSchemaHelper::sanitize);
+        } else if (node.isObject()) {
+            ObjectNode objectNode = (ObjectNode) node;
+
+            JsonNode formatNode = node.get("format");
+            if (formatNode != null &&
```

Comment:
Perhaps using that `isKnownFormat` helper method.

Reply:
Thanks, good idea!